[2024-03-29T02:03:54.320+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=download_rocket_launches/run_id=manual__2024-03-29T02:03:27.062809+00:00/task_id=notify permission to 509
[2024-03-29T02:03:54.321+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=download_rocket_launches/run_id=manual__2024-03-29T02:03:27.062809+00:00/task_id=notify permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=download_rocket_launches/run_id=manual__2024-03-29T02:03:27.062809+00:00/task_id=notify'
[2024-03-29T02:03:54.351+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=download_rocket_launches/run_id=manual__2024-03-29T02:03:27.062809+00:00/task_id=notify permission to 509
[2024-03-29T02:03:54.352+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=download_rocket_launches/run_id=manual__2024-03-29T02:03:27.062809+00:00/task_id=notify permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=download_rocket_launches/run_id=manual__2024-03-29T02:03:27.062809+00:00/task_id=notify'
[2024-03-29T02:03:54.387+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: download_rocket_launches.notify manual__2024-03-29T02:03:27.062809+00:00 [queued]>
[2024-03-29T02:03:54.398+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: download_rocket_launches.notify manual__2024-03-29T02:03:27.062809+00:00 [queued]>
[2024-03-29T02:03:54.399+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-03-29T02:03:54.414+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): notify> on 2024-03-29 02:03:27.062809+00:00
[2024-03-29T02:03:54.422+0000] {standard_task_runner.py:57} INFO - Started process 184 to run task
[2024-03-29T02:03:54.426+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'download_rocket_launches', 'notify', 'manual__2024-03-29T02:03:27.062809+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/rockets.py', '--cfg-path', '/tmp/tmpnujdkfma']
[2024-03-29T02:03:54.428+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask notify
[2024-03-29T02:03:54.488+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=download_rocket_launches/run_id=manual__2024-03-29T02:03:27.062809+00:00/task_id=notify permission to 509
[2024-03-29T02:03:54.491+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=download_rocket_launches/run_id=manual__2024-03-29T02:03:27.062809+00:00/task_id=notify permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=download_rocket_launches/run_id=manual__2024-03-29T02:03:27.062809+00:00/task_id=notify'
[2024-03-29T02:03:54.497+0000] {task_command.py:415} INFO - Running <TaskInstance: download_rocket_launches.notify manual__2024-03-29T02:03:27.062809+00:00 [running]> on host a9c2f2661b7a
[2024-03-29T02:03:54.583+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='download_rocket_launches' AIRFLOW_CTX_TASK_ID='notify' AIRFLOW_CTX_EXECUTION_DATE='2024-03-29T02:03:27.062809+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-29T02:03:27.062809+00:00'
[2024-03-29T02:03:54.585+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-29T02:03:54.585+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "There are now $(ls /tmp/images/ | wc -l) images."']
[2024-03-29T02:03:54.600+0000] {subprocess.py:86} INFO - Output:
[2024-03-29T02:03:54.614+0000] {subprocess.py:93} INFO - There are now 7 images.
[2024-03-29T02:03:54.614+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-29T02:03:54.638+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=download_rocket_launches, task_id=notify, execution_date=20240329T020327, start_date=20240329T020354, end_date=20240329T020354
[2024-03-29T02:03:54.680+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-03-29T02:03:54.699+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
